apps/mobile/src/store/accounts/utils.ts

```diff
@@ -8,10 +9,16 @@ export const accountStoreSchema = z.object({
   icon: z.string(),
   name: z.string(),
   status: z.string(),
+  type: z.string().optional(),
+  fingerprint: z.string().optional(),
+  accountIndex: z.number().optional(),
 });
 export interface AccountStore {
   id: string;
   icon: string | AvatarIcon;
   name: string;
   status: AccountStatus;
+  type?: WalletStore['type'];
+  fingerprint?: string;
+  accountIndex?: number;
```

Comment:
This data is encoded in the ID, so we don't need to actually add write it to the store, we can infer it when we read from the store.

This is already done in `initalizeAccount`

Reply:
Thanks, I'll remove. 

I had been adding this to account to lookup the address but have since replaced it with `signers.list`